pkg/image/util/helpers.go

```diff
@@ -3,18 +3,23 @@ package util
 import (
 	"encoding/json"
 	"fmt"
+	"strings"
 
 	"github.com/docker/distribution/manifest/schema1"
 	"github.com/docker/distribution/manifest/schema2"
 	"github.com/golang/glog"
 	godigest "github.com/opencontainers/go-digest"
 
+	kerrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/sets"
 
 	dockerv10 "github.com/openshift/api/image/docker10"
 	imagev1 "github.com/openshift/api/image/v1"
 	imageapi "github.com/openshift/origin/pkg/image/apis/image"
 	dockerapi10 "github.com/openshift/origin/pkg/image/apis/image/docker10"
+	imagereference "github.com/openshift/origin/pkg/image/apis/image/reference"
+	"github.com/openshift/origin/pkg/image/internal/digest"
```

Comment:
there seems to be nothing internal in this package, I would move it to pkg/image/util

Reply:
I think this `internal` package was to have a stable digest and reference implementation, not to rely on docker ones. But I'd defer to @dmage or @legionus they will know better. I just moved it to a place where the new helpers can use it.